components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -630,6 +632,16 @@ public List<String> getOrganizationsMetaAttributes(Integer limit, String after,
                                                                     expressionNodes);
     }
 
+    @Override
+    public Map<String, String> getOrganizationIdToNameMap(List<String> orgIdList)
```

Comment:
what about generalizing this method to something like `getBasicOrganizationDetailsByOrgIDs`? IMO, this service method is too specific. Also, we can have a join to UM_TENANT table and get the org handle as well.

Reply:
WDYT @AnuradhaSK ?